### PR TITLE
Slow pummel bugfixes + improvements

### PR DESCRIPTION
**Pull request by jobrien97**
Increases the time in between pummels, varying per character, but with a cap of 12f in between.

Standardizes all pummels to 2% damage.

Standardizes all pummels to a 2.5 hitlag multiplier.

Decreases base grab hold time from 90f -> 76f, and increases the multiplier for grab hold time added from total damage from 1.7 -> 2.2 (P+ values).

### CODE REVIEW

**Review by WuBoytH — DISMISSED** [review]

**Review by WuBoytH — DISMISSED** [review]

File: fighters/common/src/function_hooks/attack.rs [hunk 1]
```diff
@@ -13,14 +13,38 @@ unsafe fn attack_module_set_attack(module: u64, id: i32, group: i32, data: &mut
...
+            data.sub_shield = 0;
+            data.lr_check = smash2::app::AttackLRCheck::Pos;
+        }
+        if (*boma).is_status(*FIGHTER_STATUS_KIND_CLIFF_ATTACK) {
+            data.power = 8.0;
+            data.vector = 361;
+            data.r_eff = 50;
+            data.r_add = 65;
+            data.sub_shield = 0;
+        }
```
Comment (SuddyN) [inline fighters/common/src/function_hooks/attack.rs:40]: is this not duplicated code, as in lines 49-66 of this file? You moved the code for `FIGHTER_STATUS_KIND_DOWN_STAND_ATTACK`, but `FIGHTER_STATUS_KIND_SLIP_STAND_ATTACK` and `FIGHTER_STATUS_KIND_CLIFF_ATTACK` were duplicated

File: fighters/nana/src/acmd/other.rs [hunk 2]
```diff
@@ -346,9 +346,12 @@ unsafe fn nana_catch_turn_sound(fighter: &mut L2CAgentBase) {
...
     let boma = fighter.boma();
+    if is_excute(fighter) {
+        VarModule::on_flag(boma.object(), vars::common::status::PUMMEL_OVERRIDE_GLOBAL_STATS);
+    }
     frame(lua_state, 1.0);
     if is_excute(fighter) {
-        ATTACK(fighter, 0, 0, Hash40::new("top"), 1.0, 361, 100, 30, 0, 5.8, 0.0, 7.2, 7.4, None, None, None, 1.7, 1.0, *ATTACK_SETOFF_KIND_OFF, *ATTACK_LR_CHECK_F, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_HEAD);
+        ATTACK(fighter, 0, 0, Hash40::new("top"), 1.0, 361, 100, 30, 0, 5.8, 0.0, 7.2, 7.4, None, None, None, 1.25, 1.0, *ATTACK_SETOFF_KIND_OFF, *ATTACK_LR_CHECK_F, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_HEAD);
         AttackModule::set_catch_only_all(boma, true, false);
     }
```
Comment (SuddyN) [inline fighters/nana/src/acmd/other.rs]: Why should Nana's pummel be any different from popo's? Popo's seems fine to have on both of them

File: fighters/bayonetta/src/acmd/other.rs [hunk 3]
```diff
@@ -651,6 +651,33 @@ unsafe fn sound_appeallwl(fighter: &mut L2CAgentBase) {
...
+    frame(lua_state, 4.0);
+    if is_excute(agent) {
+        ATTACK(agent, 0, 0, Hash40::new("top"), 1.0, 361, 100, 30, 0, 5.0, 0.0, 12.0, 10.0, None, None, None, 1.25, 1.0, *ATTACK_SETOFF_KIND_OFF, *ATTACK_LR_CHECK_F, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_PUNCH);
+        AttackModule::set_catch_only_all(boma, true, false);
+    }
+    wait(lua_state, 1.0);
+    if is_excute(agent) {
+        AttackModule::clear_all(boma);
+    }
+}
```
Comment (SuddyN) [inline fighters/bayonetta/src/acmd/other.rs:679]: in-game, the damage was still 2.0 per hit. also, for some reason I got a crash during bayo's grab release vs pac man

Comment (SuddyN) [reply fighters/bayonetta/src/acmd/other.rs:679]: bayo's two hits have so much hitlag that they look like two inidivudual pummels. Could not replicate the crash

**Review by SuddyN — CHANGES_REQUESTED** [review]

File: fighters/common/src/function_hooks/attack.rs (hunk 1 above)
Comment (jobrien97) [reply fighters/common/src/function_hooks/attack.rs:40]: whoops, good catch, fixed

File: fighters/bayonetta/src/acmd/other.rs (hunk 3 above)
Comment (jobrien97) [reply fighters/bayonetta/src/acmd/other.rs:679]: I did not install the function lol

File: fighters/nana/src/acmd/other.rs (hunk 2 above)
Comment (jobrien97) [reply fighters/nana/src/acmd/other.rs]: Yeah idk, it now uses the global stats

**Review by SuddyN — APPROVED** [review]

**Pull request merged**